r-seurat-scripts-post-install-tests.sh

```diff
@@ -27,7 +30,7 @@ fi
 
 test_data_url='https://s3-us-west-2.amazonaws.com/10x.files/samples/cell/pbmc3k/pbmc3k_filtered_gene_bc_matrices.tar.gz'
 test_working_dir=`pwd`/'post_install_tests'
-test_data_archive=$test_working_dir/`basename $test_data_url`
+export test_data_archive=$test_working_dir/`basename $test_data_url`
 
 # Clean up if specified
```

Comment:
@pinin4fjords this section is suppose to be empty ?

Reply:
@suhaibMo Not sure what you're seeing as the empty section- but thanks for the review!